### DISCUSSION TITLE
Adding an external link to sidebar?

### DISCUSSION THREAD

**Discussion by anthonyshew** [fuma-nama/fumadocs#261] [Ideas]
Currently rewriting the Turborepo documentation using Fumadocs and it's incredible.

I have a small feature request for being able to add an external link to the sidebar similar to [the Nextra API](https://nextra.site/docs/docs-theme/page-configuration#external-links).

It's also possible that I could build this functionality using the primitives in fumadocs but couldn't figure it out. Is there a simple, idiomatic way that I haven't yet discovered?

Thank you again for the great work!

**Comment by fuma-nama** [comment]
I prefer adding a sidebar footer or nav link instead tho, I can extend the current Page Tree API to provide similar functionality. (Feature request in issue please)

**Reply by anthonyshew** [reply]
Apologies for coming to Discussions! Will head over to create an Issue.

**Reply by anthonyshew** [reply]
Closing now in favor of #264

**Comment by fuma-nama** [comment]
However, if you can manipulate the page tree directly, you can insert a `Item` node.
